axelor-human-resource/src/main/java/com/axelor/apps/hr/service/project/ProjectPlanningTimeServiceImpl.java

```diff
@@ -555,4 +559,100 @@ protected ProjectConfig getProjectConfig(ProjectPlanningTime projectPlanningTime
     }
     return null;
   }
+
+  @Override
+  @Transactional(rollbackOn = Exception.class)
+  public void updateProjectPlannings(Sprint sprint, List<ProjectTask> projectTaskList) {
+
+    if (!isValidSprint(sprint) || CollectionUtils.isEmpty(projectTaskList)) {
+      return;
+    }
+
+    for (ProjectTask projectTask : projectTaskList) {
+
+      if (!isValidProjectTask(projectTask)) {
+        return;
+      }
+
+      List<ProjectPlanningTime> planningTimeList = Collections.emptyList();
+
+      ProjectTask projectTaskDb =
+          Optional.ofNullable(projectTask.getId()).map(projectTaskRepo::find).orElse(null);
+
+      if (projectTaskDb != null && projectTaskDb.getSprint() != null) {
+        Employee employeeDb =
+            Optional.of(projectTaskDb.getAssignedTo()).map(User::getEmployee).orElse(null);
+        planningTimeList = fetchPlanningList(projectTaskDb.getSprint(), employeeDb, projectTask);
+      }
+
+      if (CollectionUtils.isNotEmpty(planningTimeList)) {
+        planningTimeList.forEach(
```

Comment:
For the update part, do it only if the planningTime.fromDate is > appBaseService.getTodayDate